src/loadPage.ts

```diff
@@ -10,8 +10,22 @@ export function loadPage({
   pagesDirectory: string;
   pagePath: string;
 }): NextPageFile {
+  const tmpWindow = global.window;
+
+  // Next.JS executes some rendering logic conditionally, depending if its on server.
+  // https://github.com/vercel/next.js/blob/canary/packages/next/next-server/lib/side-effect.tsx#L3
+  // We want that logic to execute, so we have to trick it to think code is executing on server
+  // Because expression is evaluated on top lelve, we have to "mock" it before module is evaluated -- before require;
+  // @ts-ignore
+  delete global.window;
```

Comment:
Hacky, but I don't see any better way BTH

Reply:
I actually like it. It makes perfectly sense since we're testing isomorphic code in a polyfilled environment. The only better solution I can think of would consist in somehow temporarily disabling JSDOM or delaying its initialisation but it seems not feasible.